src/main/java/org/jqassistant/contrib/plugin/php/scanner/PHPSourceParser.java

```diff
@@ -38,30 +37,25 @@
         this.fileDescriptor = fileDescriptor;
     }
 
+    /**
+     * parse php file
+     * @param item 
+     */
     void parseFile(final FileResource item) {
         try {
-         
             InputStream inputStream = item.createStream();
             Lexer lexer = new PhpLexer(CharStreams.fromStream(inputStream));
             TokenStream tokenStream = new CommonTokenStream(lexer);
             PhpParser parser = new PhpParser(tokenStream); 
-            //parser.setBuildParseTree(true);      
             ParseTree tree = parser.htmlDocument();
             //System.err.println(tree.toStringTree(parser));
             
             PHPFileParser a = new PHPFileParser(new Helper(store, tokenStream), this.fileDescriptor);
             a.parse(tree);
             
-//            Interval sourceInterval = tree.getChild(0).getSourceInterval();
-//            Token firstToken = tokenStream.get(sourceInterval.a);
-//            int line = firstToken.getLine();
-//            System.out.println("org.jqassistant.contrib.plugin.php.scanner.PHPSourceParser.parseFile()" + line) ;
-        
         } catch (Exception e) {
          e.printStackTrace();
-         System.out.println(e);
+            LOGGER.error("file parser error", e);
```

Comment:
Using the LOGGER is definitely better. Just please use something nicer formatted like `"Error while parsing file:"`.